pkg/sdk/testint/materialized_views_gen_integration_test.go

```diff
@@ -305,44 +305,6 @@ func TestInt_MaterializedViews(t *testing.T) {
 		assert.Equal(t, false, alteredView.IsSecure)
 	})
 
-	// Based on usage notes, set/unset tags is done through VIEW (https://docs.snowflake.com/en/sql-reference/sql/alter-materialized-view#usage-notes).
-	// TODO [SNOW-1022645]: discuss how we handle situation like this in the SDK
```

Comment:
what about this TODO?

Reply:
This is still referenced in get/set/unset tag functions. Note that these functions now use proper object types.